deploy/vagrant/conf/alluxio.yml

```diff
@@ -4,13 +4,16 @@
 # {Github | Local | Release}
 Type: Local
 
+# If the version is less than 1.1.0, ie. 1.0.1. This should be false for 1.1.0-SNAPSHOT. {true|false}
+Version_LessThan_1.1.0: false
```

Comment:
I think this could be derived programatically from the output of `bin/alluxio version`